dev-python/pygame/patches/pygame-1.9.4.patchset

```diff
@@ -302,18 +302,18 @@ index c54a93a..1b9d208 100644
  src/scale_mmx64_gcc.c
```

Comment:
Do we still need to drop these `scale` files?

Reply:
Probably.